client/jetpack-cloud/sections/manage/pricing/license-products-list/simple-license-multi-item-card.tsx

```diff
@@ -0,0 +1,113 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback, useState } from 'react';
+import MultipleChoiceQuestion from 'calypso/components/multiple-choice-question';
+import { useProductDescription } from 'calypso/jetpack-cloud/sections/partner-portal/hooks';
+import getProductShortTitle from 'calypso/jetpack-cloud/sections/partner-portal/lib/get-product-short-title';
+import getProductVariantShortTitle from 'calypso/jetpack-cloud/sections/partner-portal/lib/get-product-variant-short-title';
+import { useDispatch } from 'calypso/state';
+import { APIProductFamilyProduct } from 'calypso/state/partner-portal/types';
+import { ItemPrice } from './item-price';
+import 'calypso/my-sites/plans/jetpack-plans/product-store/simple-item-card/style.scss';
+import './style.scss';
+
+type SimpleLicenseMultiItemCardProps = {
+	items: APIProductFamilyProduct[];
+	bundleSize?: number;
+	ctaAsPrimary?: boolean;
+	ctaHref?: string;
+	moreInfoLink?: React.ReactNode;
+	isCondensedVersion?: boolean;
+	isCtaDisabled?: boolean;
+	isCtaExternal?: boolean;
+	onClickCta?: VoidFunction;
+};
+
+export const SimpleLicenseMultiItemCard = ( {
+	items,
+	bundleSize,
+	ctaAsPrimary,
+	ctaHref,
+	moreInfoLink,
+	isCtaDisabled,
+	isCtaExternal,
+	onClickCta,
+}: SimpleLicenseMultiItemCardProps ) => {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const [ item, setItem ] = useState( items[ 0 ] );
+
+	let title = item.name;
+	if ( title.startsWith( 'Jetpack Security' ) ) {
+		title.replace( 'Jetpack Security', 'Security' );
+	}
+	if ( title.startsWith( 'Jetpack VaultPress' ) ) {
+		title = 'VaultPress Backup';
+	} else {
+		title = getProductShortTitle( item, true );
+	}
```

Comment:
As above, probably can do with this alone:

```suggestion
	let title = getProductShortTitle( item, true );
```

Reply:
Done in e9dd3afd5dbbda87b34d96ab04efea54e203f064.